.travis.yml

```diff
@@ -5,7 +5,13 @@ os:
   - linux
 
 julia:
-  - 1.0
```

Comment:
Sorry for the misunderstanding - when you helped me out in fixing Travis requirements for including `hypernetx`, the Julia version in the updated `.travis.yml` file was the 1.2. I updated the Julia version in the current pull request as I thought we needed to modify it. I will restore the 1.0 version.

Reply:
Sorry for the misunderstanding - when you helped me out in fixing Travis requirements for including `hypernetx`, the Julia version in the updated `.travis.yml` file was the 1.2. I updated the Julia version in the current pull request as I thought we needed to modify it. I will restore the 1.0 version.

Reply:
If it will not work with 1.0 we should make perhaps a testing matrix with 1.0 and 1.3 and not run hypernetx related test with 1.0. In either case 1.0 compability is useful feature.